Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -41,6 +41,7 @@
 #import "WebPageProxy.h"
 #import "WebProcessProxy.h"
 #import "WindowKind.h"
+#import <QuartzCore/CATransactionPrivate.h>
```

Comment:
You can't just use SPI directly like this; you need to put what you need in QuartzCoreSPI.h (follow the existing code there).